### ISSUE TITLE
Support for Windows on Arm

### ISSUE THREAD

**Issue by pbo-linaro** [emscripten-core/emsdk#1116]
When calling emsdk.bat with an arm64 python, it fails with following error:

```
$ file $(which python)
/c/wenv/arm64/python/tools/python: PE32+ executable (console) Aarch64, for MS Windows
$ cmd.exe /c 'emsdk.bat install latest'
Resolving SDK alias 'latest' to '3.1.23'
Resolving SDK version '3.1.23' to 'sdk-releases-upstream-bfd5e63a44ba4c8568cd8ac87c27b35e40732bf4-64bit'
error: tool or SDK not found: 'sdk-releases-upstream-bfd5e63a44ba4c8568cd8ac87c27b35e40732bf4-64bit'
```

Since this platform is not yet supported, this workaround (https://github.com/emscripten-core/emsdk/pull/1115) could be used.

Beyond that, it would be nice to offer a native toolchain for this new platform.

**Comment by Blackhex** [comment]
JFYI, we have work on providing the pre-build Windows Arm64 binaries in progress. The first upstream PR regarding this can be found at #1186.

**Comment by pbo-linaro** [comment]
It's good to see this support coming :+1:

**Comment by sbc100** [comment]
@dschuff just reviving this old issue for 2023.

Should we start shipping windows arm64 binaries soon, or is emulation the best option still (#1115)

**Comment by dschuff** [comment]
I wouldn't mind seeing this, but it would be pretty unfortunate if we had no way to test it. It looks like there are github hosted runners avaiable now?

**Comment by sbc100** [comment]
Yes, it looks like we could need to use github actions and now circleci for this.

**Comment by sbc100** [comment]
How hard would the cross compile on windows be ?

**Comment by dschuff** [comment]
I don't think it would be too bad? Probably just change the CMake platform (and maybe the MSVC environment variables, if the actual compiler is different?)
